include/revng/Pipes/StringMapContainer.h

```diff
@@ -29,17 +33,22 @@ class StringMapContainer : public pipeline::Container<StringMapContainer> {
 private:
   MapType Map;
   const pipeline::Kind *TheKind;
+  const model::Binary *Model;
 
 public:
   static char ID;
 
 public:
   StringMapContainer(llvm::StringRef Name,
                      llvm::StringRef MIMEType,
-                     const pipeline::Kind &K) :
+                     const pipeline::Kind &K,
+                     const model::Binary *Model) :
```

Comment:
Can this be `nullptr`? If no, use references.

Reply:
It can be if you don't use `enumerate`. But for now we don't have any place where we need to pass a `nullptr` to it, nor I have in mind cases in the near future where we might wan to do that. I'll switch to references for now.

Reply:
Done!